### PR TITLE
Added `opencensus` to known list

### PR DESCRIPTION
**Pull request by pritishpai**
## Changes
Added `opencensus` to known list 

### Linked issues
Partly resolve #1931

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -12653,6 +12653,12 @@
     "openai.types.shared_params.function_parameters": [],
     "openai.version": []
   },
+  "opencensus": {
+    "opencensus": []
+  },
+  "opencensus-context": {
+    "opencensus": []
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:12660]: how come that two different libraries "own" the same root package?

**Review by nfx — DISMISSED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (pritishpai) [reply src/databricks/labs/ucx/source_code/known.json]: opencensus-context is a smaller focused subset of opencensus.

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -33798,5 +33801,7 @@
     "zipp.compat.py310": [],
     "zipp.glob": []
   },
-  "zstd": {}
+  "zstd": {
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:33804]: Please remove this change

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
Minor change

File: src/databricks/labs/ucx/source_code/known.json [hunk 3]
```diff
@@ -33797,6 +33800,5 @@
     "zipp.compat": [],
     "zipp.compat.py310": [],
     "zipp.glob": []
-  },
-  "zstd": {}
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:33801]: Do not remove this

**Review by JCZuurmond — APPROVED** [review]
LGTM

**Pull request merged**